src/bindgen/index.md

```diff
@@ -20,6 +20,8 @@ use Rust in a C/C++ codebase in the first place!
 
 ## Getting Set Up
 
+> **Note:** Bindgen **requires** clang. Installation instructions can be found [here](https://github.com/rust-lang-nursery/rust-bindgen/blob/master/book/src/requirements.md#installing-clang-39)
```

Comment:
Did you end up figuring out why highlighting wasn't working? From that dev console output it almost looks like there's a bug in the library mdbook uses for parsing and rendering markdown.